src/ameba/config.cr

```diff
@@ -57,10 +58,18 @@ class Ameba::Config
     Path[XDG_CONFIG_HOME] / "ameba/config.yml",
   }
 
-  DEFAULT_GLOBS = %w(
-    **/*.cr
-    !lib
-  )
+  {% if compare_versions(Crystal::VERSION, "1.15.0") >= 0 %}
```

Comment:
This could use the helper method as well, right?

Reply:
Yeah if I change it to appending to the array after it's created

Reply:
If we want to use `Ameba.ecr_supported?` here, I can replace this with:
```cr
  DEFAULT_GLOBS = %w(
    **/*.cr
    !lib
  )

  if Ameba.ecr_supported?
    DEFAULT_GLOBS << "**/*.ecr"
  end
```